content/departments/product-engineering/engineering/cloud/delivery/deployment/helm.md

```diff
@@ -48,3 +48,117 @@ In the event you need assistance from Delivery, please gather the following info
+
+Some upgrades may include a new deployment, and you would want to review any configuration related to that and customize it as necessary.
+
+In contrast, upgrading Sourcegraph using kustomize meant first merging the release changes from the upstream Sourcegraph repository and resolving any merge conflicts that occurred.
+
+#### Can our Helm chart be forked?
+
+Yes, although we strongly recommend against it. If a customer does fork the chart and make their own customizations, they’ll be responsible for keeping the fork in sync with the official chart and managing any merge conflicts.
+
+Instead, we do support two methods of configuration that offer an alternative to forking the repo, using Kustomize on top of Helm, or creating a sub chart. [Read more about both here](https://docs.sourcegraph.com/admin/install/kubernetes/helm#advanced-configuration).
+
+If you learn that the Sourcegraph chart is missing functionality a customer needs, please tell us about it (or submit a PR) so their feature can be added and they won’t need to maintain a separate chart yourself. (Even if they’re comfortable with one of the two approaches covered in the link above.
+
+#### Can you output the manifest files from the Helm chart?
+
+The helm template command can be used to render Kubernetes manifests from the chart. This output can then be inspected, checked into source control, or applied to the cluster via kubectl.
+
+More about this in the [Reviewing changes](https://docs.sourcegraph.com/admin/install/kubernetes/helm#reviewing-changes) section of our Helm docs.
+
+#### What are the limitations of using Helm?
+
+Without opting to either use Kustomize on top of Helm, or create a sub chart ([read more about both here](https://docs.sourcegraph.com/admin/install/kubernetes/helm#advanced-configuration)), customers are limited to the configuration options built into the chart by Sourcegraph.
+
+If a customer needs additional features not included in the chart, it needs to be requested to be added.
+
+#### Do we support converting Docker Compose/Kustomize/plain Kubernetes to use Helm?
+
+Not yet, but it’s planned for FY23-Q2/3.
+
+Instead, it should be used on a new installation if a customer really wished to move to Helm.
+
+If a customer wishes to try and convert themselves, any customizations will need to be converted to a override file ([see here for more info](https://docs.sourcegraph.com/admin/install/kubernetes/helm#configuration)). The [sourcegraph/examples folder](https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-helm/-/tree/charts/sourcegraph/examples) in the deploy-sourcegraph-helm repo has some samples of commonly modified values that can be used as a starting point.
```

Comment:
We know there are breaking changes between helm and kustomize (especially if they aren't already using the non-privileged overlay). Might be worth a stronger warning here.

Reply:
second this.

We've done the migration from kustomize -> helm, the process is not easy.